app/assets/javascripts/leaflet.map.js

```diff
@@ -347,15 +347,22 @@ L.OSM.Map = L.Map.extend({
   },
 
   setSidebarOverlaid: function (overlaid) {
-    var sidebarWidth = 350;
+    var mobileDeviceWidth = window.getComputedStyle(document.documentElement).getPropertyValue("--bs-breakpoint-md");
+    var isMobileDevice = window.matchMedia(`(max-width: ${mobileDeviceWidth})`).matches;
```

Comment:
I wonder if `mobileDevice` is the right way to describe this - what we're really looking at is  which side of bootstrap's medium breakpoint we are which in turn controls whether "sidebars" go at the side or the top. Maybe @gravitystorm has a thought?

Also I think technically to match https://getbootstrap.com/docs/5.0/layout/breakpoints/#max-width we should subtract `.02` from max-width in the match?

Reply:
> Maybe @gravitystorm has a thought?

In general we must avoid referring to narrow screens as "mobile devices" - that causes endless confusion among our developers. Mobile devices can have large screen widths, desktop windows can become narrow, etc.

With bootstrap the `md` means "medium" not "mobile device" - perhaps this is the source of the confusion here?

https://getbootstrap.com/docs/5.3/layout/breakpoints/#available-breakpoints